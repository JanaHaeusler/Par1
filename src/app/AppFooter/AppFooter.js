import styled from 'styled-components/macro'
import Navigation from '../Navigation'

export default function AppFooter({className}) {

  return (
    <FooterStyled className={className}>
      <Navigation/>
    </FooterStyled>
  )
}

const FooterStyled = styled.footer`
  margin: 0 auto;
  display: flex;
  justify-content: space-evenly;
  align-items: center;
  height: 50px;
  max-width: 1240px;
  box-shadow: 0 0 10px var(--primary-dark);
  border-radius: 25px 25px 0 0 ;
  background: var(--gradient-dark);
`