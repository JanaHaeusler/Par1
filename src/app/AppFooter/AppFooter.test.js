import { render } from '@testing-library/react'
import { BrowserRouter as Router } from 'react-router-dom'
import AppFooter from './AppFooter'

describe('AppFooter', () => {
  it('renders correctly', () => {
    const { container } = render(
      <Router>
        <AppFooter />
      </Router>
    )
    expect(container.firstChild).toMatchSnapshot()
  })
})
