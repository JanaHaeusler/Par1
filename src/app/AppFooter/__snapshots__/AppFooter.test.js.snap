// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppFooter renders correctly 1`] = `
.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: space-evenly;
  -webkit-justify-content: space-evenly;
  -ms-flex-pack: space-evenly;
  justify-content: space-evenly;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: 100%;
}

.c2 {
  width: 28px;
  height: 28px;
  fill: var(--text-light);
  opacity: 40%;
}

.c2.active {
  opacity: 100%;
}

.c0 {
  margin: 0 auto;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: space-evenly;
  -webkit-justify-content: space-evenly;
  -ms-flex-pack: space-evenly;
  justify-content: space-evenly;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  height: 50px;
  max-width: 1240px;
  box-shadow: 0 0 10px var(--primary-dark);
  border-radius: 25px 25px 0 0;
  background: var(--gradient-dark);
}

<footer
  class="c0"
>
  <nav
    class="c1"
  >
    <a
      aria-current="page"
      class="c2 active"
      data-testid="button-home-page"
      href="/"
    >
      <svg>
        home.svg
      </svg>
    </a>
    <a
      class="c2"
      data-testid="button-form-page"
      href="/create"
    >
      <svg>
        plus.svg
      </svg>
    </a>
  </nav>
</footer>
`;
